### ISSUE TITLE
Loading Spinner of visualization components should block the canvas only

### ISSUE THREAD

**Issue by BeritJanssen** [CentreForDigitalHumanities/I-analyzer#1394]
**Is your feature request related to a problem? Please describe.**
Right now, requesting a visualization from the backend will display the loading spinner, but this will block not only the canvas, but also input elements controlling normalization etc. This means the user cannot interact with controls anymore while the graph is loading, effectively making it impossible to cancel the running tasks.

**Describe the solution you'd like**
It would be better if the loading spinner only blocked the canvas, so the user could keep interacting with the controls, which could cancel the running request for visualization data to the backend.

**Comment by lukavdplas** [comment]
Blocking the options was sort-of intentional, as it prevents the complication of cancelling one subscription and starting another. That's not impossible to handle, though

See also #895

**Comment by BeritJanssen** [comment]
Yes, I opened this issue exactly to address that issue: interacting with the controls will abort the tasks then.

**Comment by BeritJanssen** [comment]
This has been solved.
